include/aspect/postprocess/tracer.h

```diff
@@ -35,36 +38,125 @@ namespace aspect
     template <int dim>
     class PassiveTracers : public Interface<dim>, public ::aspect::SimulatorAccess<dim>
```

Comment:
I will. Would you be ok, if I change the class name to `Tracers`? For one that is the name of the input parameter anyway, and also they are technically not far away from being used for *active* purposes as well, if a user wants to do so.